### DISCUSSION TITLE
[SANDBOX] Boutique incomplete

### DISCUSSION THREAD

**Discussion by thefox82** [PayGreen/documentation#4] [Q&A] [answered]
Bonjour,
J'essaye de créer une boutique en SANDBOX mais j'ai le message d'erreur:
Certaines informations de votre boutique sont manquantes. Merci de les compléter :
* Global
* Offre de facturation
Mais je ne comprends pas à quel paramètre cela fait référence.
Merci

**Comment by AdrienBr** [comment, marked as answer]
Bonjour @thefox82 ,
En effet cette information n'est pas nécessaire en Sandbox, elle ne sera plus demandée lors d'une prochaine release.

Bonne journée,
Adrien
